modules/lead/product-base/production.tf

```diff
@@ -26,64 +26,6 @@ resource "helm_release" "production_product_init" {
   provider  = helm.production
 }
 
-resource "kubernetes_role" "jenkins_production_role" {
```

Comment:
we actually need to keep this in the base module since we will always need some sort of role some CI (whether its Jenkins or codepipeline) to make changes.  So keep this here, but name the name more generic so it could apply to either Jenkins or CodePipeline